zone/bonuses.cpp

```diff
@@ -673,20 +673,7 @@ void Mob::AddItemBonuses(const EQ::ItemInstance* inst, StatBonuses* b, bool is_a
 		}
 
 		if (item->ExtraDmgAmt != 0 && item->ExtraDmgSkill <= EQ::skills::HIGHEST_SKILL) {
-			if (item->ExtraDmgSkill == ALL_SKILLS) {
```

Comment:
This was necessary to make sure this doesn’t affect all skills.

Reply:
oops, didn't mean to remove that. added back